README.md

```diff
@@ -29,6 +29,21 @@ To start Jekyll server locally.
 5. Make some changes, refresh your browser to preview them.
 6. Submit a pull request.
 
+## New release
+
+For each new release we keep stable documentation tree for the record.
+
+### Foreman Core
+
+1. cp -r manuals/X.Y manuals/X.Z
+2. edit file _layouts/manual.html and modify "latestversion"
```

Comment:
No, it's OK I think.  I'll leave @stbenjam to merge if he likes, I can't comment on the plugin docs much.